packages/components/src/Icons/stories/index.stories.tsx

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
Why aren't we using the default storybook configuration in our other repos?

Reply:
I'm not too sure, I believe it was apart of our package splitting internally. 
in the `main.js` we point to stories explicitly in our components with: 
```
module.exports = {
	stories: ['../../components/**/stories/index.tsx'],
	...otherConfigOptions
}
```

Reply:
I believe in Storybook 7.0 this opens up a whole new can of worms. See the following:

https://github.com/storybookjs/storybook/issues/20478
https://sheriffmoose.medium.com/storybook-markdown-docs-not-mdx-part-2-757463fcad84

I believe this is because of the creation of the concept of StoryIndexers, which are loaded based on a regex. By default the CSF loader is assigned to the stories prefix. When we remove that prefix from the glob, it no longer matches to the StoryIndexer. In order to implement this, we would need to take an approach similar to the second link (where the author implements a custom StoryIndexer to handle markdown files, but instead of a custom one we would find a way to reuse the original csf) I am not sure it is worth the trouble of diverging from the default configuration in this case. @trystendsmyth bringing this up to your attention as well because of our conversation on Friday.

Reply:
Based on @builtbyproxy 's comments in yesterdays standup, and @trystendsmyth 's comments here: https://calyptia.slack.com/archives/C04LDS4NKJ4/p1686072885408659 I am going to mark this as resolved.